x-pack/plugins/fleet/public/components/enrollment_instructions/standalone/index.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { CommandsByPlatform } from '../../../applications/fleet/sections/agents/agent_requirements_page/components/install_command_utils';
+import type { K8sMode } from '../../../components/agent_enrollment_flyout/types';
+
+export const StandaloneInstructions = (
+  kibanaVersion: string,
+  isK8s?: K8sMode
+): CommandsByPlatform => {
+  const KUBERNETES_RUN_INSTRUCTIONS = 'kubectl apply -f elastic-agent-standalone-kubernetes.yaml';
+
+  const STANDALONE_RUN_INSTRUCTIONS_LINUX = `curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-${kibanaVersion}-linux-x86_64.tar.gz
+  tar xzvf elastic-agent-${kibanaVersion}-linux-x86_64.tar.gz
+  sudo ./elastic-agent install`;
+
+  const STANDALONE_RUN_INSTRUCTIONS_MAC = `curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-${kibanaVersion}-darwin-x86_64.tar.gz
+  tar xzvf elastic-agent-${kibanaVersion}-darwin-x86_64.tar.gz
+  sudo ./elastic-agent install`;
+
+  const STANDALONE_RUN_INSTRUCTIONS_WINDOWS = `wget https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-${kibanaVersion}-windows-x86_64.zip -OutFile elastic-agent-${kibanaVersion}-windows-x86_64.zip
+  Expand-Archive .\elastic-agent-${kibanaVersion}-windows-x86_64.zip
+  .\\elastic-agent.exe install`;
```

Comment:
Can you add the missing `cd` command that was fixed for manual instructions here: https://github.com/elastic/kibana/pull/129051/files

Reply:
thanks, I missed this!